Source/BugsnagCrashReport.m

```diff
@@ -478,7 +499,11 @@ - (NSDictionary *)toJson {
     BSGDictSetSafeObject(event, BSGFormatSeverity(self.severity), BSGKeySeverity);
     BSGDictSetSafeObject(event, [self breadcrumbs], BSGKeyBreadcrumbs);
     BSGDictSetSafeObject(event, metaData, BSGKeyMetaData);
-    
+
+    if ([self isIncomplete]) {
+        BSGDictSetSafeObject(event, @YES, BSGKeyIncomplete);
```

Comment:
Do we want to include the field if it's `false` also? This would allow filtering on the value either way